.travis.yml

```diff
@@ -16,4 +16,4 @@ script: |
 
 notifications:
   email:
-    - mark.nunberg@couchbase.com
+    - stephen.tunney@gmail.com
```

Comment:
Funny. I committed this after creating the PR so my Travis build for my fork would report to me ;)